modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -53,6 +34,47 @@ spec:
 <1> Specifies the machine config pool label. 
 <2> Specifies the container runtime to deploy.
 
+* Optional: Create or edit an existing Security Context Constraint (SCC) to configure the `userNamespaceLevel` setting. This setting is optional, because the default value allows, but does not require, the use of user namespaces.
++
+.Example `SecurityContextConstraints` object definition
+[source,yaml]
+----
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: scc-admin
+userNamespaceLevel: RequirePodLevel <1>
+allowPrivilegedContainer: true
+# ...
+seLinuxContext: <2>
+  type: RunAsAny
+# ...
+runAsUser:
+  type: RunAsAny <3>
+users: []
+# ...
+----
+<1> Specify one of the following values:
+** If set to the `AllowHostLevel` value, you can set `hostUser` to `true` or `false`. `AllowHostLevel` is the default. This setting allows you to use the SCC with pods that run in the host or an individual user namespace.  
```

Comment:
> This setting allows you to use the SCC with pods that run in the host or an individual user namespace.

subject and object should switch: When this field is set as such and pods are running under this SCC, pods are allowed to run with or without a user namespace